contents/docs/libraries/python.mdx

```diff
@@ -245,12 +245,6 @@ POSTHOG_DJANGO = {
 }
 ```
 
-# Development
-
-## Naming Confusion
-
-As our open source project [PostHog](https://github.com/PostHog/posthog) shares the same module name, we created a special `posthoganalytics` package, mostly for internal use to avoid module collision. It is the exact same.
```

Comment:
I think this is relevant, and makes sense? Specially helps clarify why we have two `setup.py`s in posthog-python?

Reply:
I was proactive on the user report and didn't double check, indeed still relevant!